### ISSUE TITLE
8.5.1 regression: `cargo watch -- cargo clippy`

### ISSUE THREAD

**Issue by kpcyrd** [watchexec/cargo-watch#298]
I've tested the latest release and noticed there seems to be some kind of regression:

```
% cargo watch -- cargo clippy
error: The subcommand 'clippy' wasn't recognized
	Did you mean 'clippy'?

If you believe you received this message in error, try re-running with 'cargo watch -- clippy'

USAGE:
    cargo watch [FLAGS] [OPTIONS]

For more information try --help
```

I can work around it with `cargo watch -x clippy` but this update is likely breaking many workflows. Interestingly, this still works:

```
cargo watch -- cargo run -- asdf
```

but this does not:

```
cargo watch -- cargo run -- help
```

**Comment by passcod** [comment]
What the heck. This is the weirdest clap misfeature.

**Comment by passcod** [comment]
Ah, I think I've found the right combination of options to make it work. Released as 8.5.2, let me know if that fixes it for you.

**Comment by kpcyrd** [comment]
Thanks for the quick bugfix release! :) I've uploaded 8.5.2 to [Arch Linux](https://archlinux.org/packages/extra/x86_64/cargo-watch/) now.
